src/setup_payload/python/SetupPayload.py

```diff
@@ -58,14 +58,15 @@ class CommissioningFlow(enum.IntEnum):
 
 
 class SetupPayload:
-    def __init__(self, discriminator, pincode, rendezvous=4, flow=CommissioningFlow.Standard, vid=0, pid=0):
+    def __init__(self, discriminator, pincode, rendezvous=4, flow=CommissioningFlow.Standard, vid=0, pid=0, include_vid_pid=False):
```

Comment:
As per spec (section 5.1.4.1), vid/pid shall be included only if commissioning flow is non standard.

BTW, what use case are we targeting?